go/lib/infra/messenger/quic_response_writer.go

```diff
@@ -135,8 +138,26 @@ func (rw *QUICResponseWriter) SendHPCfgReply(ctx context.Context, msg *path_mgmt
 	return rw.sendMessage(ctx, ctrlPld)
 }
 
+func (rw *QUICResponseWriter) SendMSRep(ctx context.Context, msg *ms_mgmt.Pld,
+	messageType infra.MessageType) error {
```

Comment:
For the QUIC response a message type is not necessary. But it is used if the connection used the UDPResponseWriter which then calls the infra.Messenger function that needs a message type. But because both of them implement the infra.ResponseWriter interface I had to add the parameter.